documentation/jetty-documentation/src/main/java/org/eclipse/jetty/docs/programming/server/session/SessionDocs.java

```diff
@@ -72,14 +75,44 @@ public void defaultSessionIdManagerWithHouseKeeper()
             e.printStackTrace();
         }
     }
+
+    public void coreSessionHandler()
+    {
+        try
+        {
+            //tag:coresession[]
+            Server server = new Server();
+            org.eclipse.jetty.session.SessionHandler sessionHandler = new org.eclipse.jetty.session.SessionHandler();
+            sessionHandler.setSessionCookie("SIMPLE");
+            sessionHandler.setUsingCookies(true);
+            sessionHandler.setUsingURLs(false);
+            sessionHandler.setSessionPath("/");
+            server.setHandler(sessionHandler);
+            sessionHandler.setHandler(new Handler.Abstract()
+            {
+                @Override
+                public boolean handle(Request request, Response response, Callback callback)
+                {
+                    Session session = request.getSession(false);
+                    Content.Sink.write(response, true, "Session=" + session.getId(), callback);
+                    return true;
+                }
+            });
+            //end::coresession[]
+        }
+        catch (Exception e)
+        {
+            e.printStackTrace();
+        }
+    }
     
-    public void servletContextWithSessionHandler()
+    public void ee10ServletContextWithSessionHandler()
```

Comment:
We only document the latest EE version, so ee10 here is implicit and you can drop it.